src/main/flight/pid.c

```diff
@@ -1337,9 +1337,9 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 
     float agGain = dT * itermAccelerator * AG_KI;
 
-    // gradually scale back integration when above windup point
+    // gradually scale back integration when above windup point on yaw only
     float dynCi = dT;
-    if (itermWindupPointInv > 1.0f) {
+    if (axis == FD_YAW && itermWindupPointInv > 1.0f) {
```

Comment:
Will have to be done differently - `axis` isn't defined here as in the current implementation `dynCi` applies to all axes.